src/metabase/models/humanization.clj

```diff
@@ -82,10 +83,15 @@
   :visibility :settings-manager
   :audit      :raw-value
   :getter     (fn []
-                (let [strategy (setting/get-value-of-type :keyword :humanization-strategy)]
-                  ;; actual advanced method has been excised. Use `:simple` instead if someone had specified
-                  ;; `:advanced`.
-                  (if (= strategy :advanced)
-                    :simple
-                    strategy)))
+                (let [strategy (setting/get-value-of-type :keyword :humanization-strategy)
+                      valid-values (set (keys (methods u.humanization/name->human-readable-name)))
+                      valid-strategy? (contains? valid-values strategy)
```

Comment:
nit: sets can be used as functions which look up their value, or return `nil` if the value doesn't exist. I think it's actually slightly more idiomatic in clojure to just do `(valid-values strategy)` here, since any non-nil value will be truthy.